families-api/README.md

```diff
@@ -0,0 +1,71 @@
+# families-api
+
+## Decisions checklist
+
+### Local development
+
+**`just dev`**
+
+- [x] To run the service locally, you should run `just dev`
+- [x] We use docker compose to start up local services
+- [x] For python code, we should also be able to run the framework's (FastAPI) native dev environment (`fastapi dev app/main.py`)
+- [x] Data for seeing the service locally should be pre-loaded and prod-like
+- [x] Any changes to code should be reflected immediately in the running service, no reloads needed
+
+**`just test`**
+
+- [ ] All tests should run when running `just test`
+- [ ] Testing should be stateless and predictable at every run
+- [ ] Tests should run the same locally and in CI
+- [ ] Local tests should run on a watch
+
+**`just build`**
+
+**`just deploy`**
+
+- [ ] the `main` branch will always be what's on production
+
+**infra**
+
+```bash
+cd infra
+pulumi up --stack production
+```
+
+## TODO
+
+### Python App
+
+**these can / shoudl be added when we have another service in to ensure we get the right abstraction**
+
+- [ ] JSON logging
+- [ ] OTel implementation
+
+## Potential middle-game
```

Comment:
@olaughter @henry-cpr because we've been talking about this and @katybaulch as you're doing some work with infra and we've spoken about microservices

I've tried to capture a near-end game for having a few isolated services running. They should be easy enough to spin up - and also then we have some simple things where we can start talking about things like

- "How do we use datalake for this?" (the passages app should give us a lead)
- "What about API access control" once we have the API we can try draw it into this graph e.g. Cloudwatch edge lambda on the cloudfront origins

All based on - let's get some simple things write and then start to compose around them - or, if we really really need to, add some more simple things.